src/controller/org.controller/index.js

```diff
@@ -486,14 +478,13 @@ router.get('/org/:shortname/user/:username',
   /*
   #swagger.tags = ['Users']
   #swagger.operationId = 'userSingle'
-  #swagger.summary = "Retrieve a user record"
+  #swagger.summary = "Retrieves a user entry for the specified username and organization short name (accessible to all users)"
   #swagger.description = "
         <h2>Access Control</h2>
-        <p>No roles needed to access the endpoint</p>
+        <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
-        <p><b>Secretariat:</b> Can see the information of a user of any Organization</p>
-        <p><b>Admin User:</b> Can only see the information of a user that belongs to the same Organization</p>
-        <p><b>Regular User:</b> Can only see the information of a user that belongs to the same Organization</p>"
+        <p><b>Regular and Admin Users:</b> Retrieves a user's information for a user in the same organization</p>
```

Comment:
Make this Regular, CNA & Admin Users. Maybe reword to "Retrieves information for a user in the same organization"